composables/transaction/transactionBurn.ts

```diff
@@ -19,7 +21,20 @@ import {
   getApiCall,
 } from '@/utils/gallery/abstractCalls'
 
+function execBurnEvm(item: ActionConsume, executeTransaction: ExecuteTransaction) {
+  executeTransaction({
+    address: item.collectionId,
+    abi: GENSOL_ABI,
+    functionName: 'burn',
+    arg: [item.nftSn],
+  })
+}
+
 export function execBurnTx(item: ActionConsume, api, executeTransaction) {
+  if (item.urlPrefix === 'base') {
```

Comment:
imx ?

Reply:
Added but I have not tested it with imx yet.